### DISCUSSION TITLE
WalletConnect v3 Deeplink

### DISCUSSION THREAD

**Discussion by amit-atr958** [WalletConnect/walletconnect-monorepo#4165] [Developer Support]
hi @glitch-txs
 For WalletConnect v3, I aim to implement deep linking. When users open my website URL in a standard browser and click on MetaMask or Trust Wallet through WalletConnect, I want to seamlessly open my site in the MetaMask DApps browser or Trust Wallet browser. I seek to replicate the same functionality that is present when interacting with Coinbase Wallet. How can I achieve this?

**Comment by glitch-txs** [comment]
hi, you can use this, and add the deeplink for the in-app browser, you can find that in each wallet's documentation
https://docs.walletconnect.com/web3modal/react/options#customwallets

**Comment by mujee7** [comment]
hi @glitch-txs I have created a custom coinbase wallet to make a deeplink. It opens the website in coinbase wallet but when I try to connect in the app it still accessing and trying that link and opens the app store.

**Reply by mujee7** [reply]
{
      id: 'CoinBase',
      name: 'CoinBase',
      image_url: 'https://upload.wikimedia.org/wikipedia/commons/thumb/3/36/MetaMask_Fox.svg/1200px-MetaMask_Fox.svg.png', // Optional
      mobile_link: 'https://go.cb-w.com/dapp?cb_url=https%3A%2F%2Ftesting-lake-two.vercel.app%0A', // Optional - Deeplink or universal
      desktop_link: 'https://go.cb-w.com/dapp?cb_url=https%3A%2F%2Ftesting-lake-two.vercel.app%0A', // Optional - Deeplink
      app_store: 'https://apps.apple.com/us/app/coinbase-buy-bitcoin-ether/id886427730', // Optional
      play_store: 'https://play.google.com/store/apps/details?id=com.coinbase.android&hl=en&gl=US' // Optional
 },

**Reply by glitch-txs** [reply]
Coinbase doesn't support the WalletConnect protocol

**Reply by mujee7** [reply]
@glitch-txs I have tried with metamask aswell, it opens the website in wallet but in app when I try to connect it still trying to access that link and opens another tab.

**Reply by glitch-txs** [reply]
that field is for testing, you should use this instead: https://docs.walletconnect.com/web3modal/react/options#featuredwalletids
